src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
+      margin: 0 10px;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+
+    font-weight: 500;
```

Comment:
There is difference between figma layout and coded layout with font-weight, sizes and color in navigation menu and card fonts. Please use pixel-perfect tool and adjust styling to match figma layout. 

Coded layout:
![image](https://github.com/elwirra/layout_catalog-test_task/assets/2406738/e63359a7-2c56-439a-a796-dfe94f95f848)
Figma:
![image](https://github.com/elwirra/layout_catalog-test_task/assets/2406738/2074b1d4-13c6-49b1-9256-f685d8ed18c5)